File: src/algorithms/repeated-character.js
/**
 * Returns the first letter from the string parameter s that appears twice.
 * 
 * @param {string} s - A string of lowercase English letters
 * @returns {string} - Returns the first string character that appears twice
 */
function repeatedCharacter (s) {
  let chars = new Set();
  for (const char of s) {
    if (chars.has(char)) {
      return char;
    } else {
      chars.add(char);
    }
  }
}

File: src/algorithms/repeated-character.py
def repeated_character(s):
    """Given a string s consisting of lowercase English letters, return the
    first letter to appear twice.

    Args:
        s (String): A string of lowercase English letters.

    Returns:
        String: Returns the first character to appear twice in s.
    """
    seen = set()
    for i in range(len(s)):
        if s[i] in seen:
            return s[i]
        seen.add(s[i])
